agent/acs/handler/acs_handler_test.go

```diff
@@ -202,13 +204,14 @@ func TestHandlerReconnectsOnConnectErrors(t *testing.T) {
 	mockWsClient.EXPECT().Close().Return(nil).AnyTimes()
 	gomock.InOrder(
 		// Connect fails 10 times
-		mockWsClient.EXPECT().Connect().Return(io.EOF).Times(10),
+		mockWsClient.EXPECT().Connect(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil, io.EOF).Times(10),
```

Comment:
Could we pass in `metrics.ACSDisconnectTimeoutMetricName` (if passing in metric name to the Connect method is deemed necessary), `wsclient.WSclientDisconnectTimeout`, and  `wsclient.WSclientDisconnectJitterMax`, instead of `gomock.Any()`, `gomock.Any()`, and `gomock.Any()` here to be more specific? (this comment applies to other lines of this file where the same change is made as well)

Reply:
I don't think we need to do that. It depends on what we are trying to test. If the test is not specifically testing the timeout or metric emitting, then I think it is unimportant what value we pass. That's what i consider the beauty of using gomock. Let me know if you think otherwise and consider it to be important to send specific values.

Reply:
Apologies, I should have marked my original comment here as a nit. IMO if we are certain of what will/should be passed in, then we should specify it to make for a stronger/more strict tests. In cases where we do not have such certainty, we could use `gomock.Any()`.

Again, this is just personal preference for me though and is non-blocking.